### ISSUE TITLE
Decision Tools Backlog Grooming

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#20068]
# Decision Tools: Grooming Document

<details><summary>About this meeting</summary>

- Wednesdays at 3:30 PM EST
- Meeting owner: Marci McGuire (PM-Decision Tools)
- Facilitator: Marci McGuire 
- Standing agenda: 
  - Add and remove Backlog Items
  - Change priority of items  
  - Change Acceptance Criteria 
  - Provide clarity on any item as requested 
  - Clarity Requests from team
  - Planned PTO
</details>

<details><summary>Backlog Grooming Prep</summary>

- Review Backlog
- Review Icebox
- Make sure new feature requests are attached to the appropriate prioritization epic
</details>

### Discussion Topics / Grooming 
- Grooming time moving to 3:30 ET starting next Sprint
- Upcoming work in authenticated space - Samara Strauss and Kevin Hoffman will be the VA POs.  Work/timing TBD.
- Any new work not already in our backlog? 
- PDF modal will continue into next Sprint
    - New request in to content for tweaks following today's review at Platform open office hours
    - Re-opened the mock-up ticket for next Sprint
    - Will ask Angela to review for accessibility as-is on Staging

- PDF audit -- will send draft slide deck out by EOD tomorrow
- Zach's thoughts on starting autocomplete/typeahead for Yellow Ribbon?

### Discussion Topics / Grooming 

- PTO - Cassandra Sept 25, Zach - Oct 14, possibly 15.  Marci possibly Oct 4.

- **Find a VA Form** PDF research
    - Working on adding alert - have mockup; next need content
    - Autocomplete?? Programmatically possible? How to prioritize?

- **Yellow Ribbon** needs improvements because it doesn't find anything for school abbreviations
    - Autocomplete?? Programmatically possible? How to prioritize?
    - We have a 44% refinement rate, and only a 9% click rate
    - Discovery POC for ranking/ordering search results - Zach can use some of Lance's time for Ruby consult.
    - Search by radius (requested by stakeholders as valuable) -- may be able to pick Lance Sanchez's brain -- BE on Facilities - sprint 58 ish

- **Other Discussion**
    - Cassandra has a lot on her plate right now
    - Quarterly/annual planning meeting that includes Dave (or a new Product Owner)
    
- **Discharge Upgrade Wizard** - Hold

### Discussion Topics / Grooming 

- **Find a VA Form** PDF research
    - Schedule meeting with Martha Wilkes, get 508 office involved to review forms
    - Add instructions to Find a VA Form  -- which version? 
    - Top 10 forms experiencing issues observed in study
- **Yellow Ribbon** needs improvements because it doesn't find anything for school abbreviations
    - We have a 44% refinement rate, and only a 9% click rate
- **Discharge Upgrade Wizard** - Still concerned about how requested changes will be detrimental to the user experience; blocked pending a path forward.

### Discussion Topics / Grooming 
- Discharge Upgrade Wizard - Cassandra suggested meeting with Content before doing mock-ups
- We are meeting with stakeholders on Friday; their response to concerns about the user experience may influence the changes.
- What can we work on until we can meet with the Discharge Upgrade Wizard folks and a new PO can help us prioritize and re-engage the Yellow Ribbon stakeholders?

### Discussion Topics / Grooming
- Discharge Upgrade Wizard
    - Content changes for Discharge Upgrade Wizard - if we don't think these changes provide a good user experience, what are our options? 
    - Note for Zach in a new Discovery issue - item on request 2 only applies to RMB.
- Go To Online Tool 
    - Which order? _Go To Online Tool_ > Spanish OR [New Text] > Spanish
    - Marci will get with Angela and Josh
 - Update ticket for PDF download - we will test with the blue version of an info link.

= 

### Discussion Topics / Grooming
- Priority for Sprint 49 - Original goal was to stay focused on Find a Form changes in order to have it ready for go-live first week of July
- Discharge Upgrade Wizard - Cannot do a kickoff & get feedback on Collab Cycle requirements until we have all the content update requests from DoD and can estimate, determine if it will impact user flow, design.  Are we going to do Alexander's as a one-off? 

### Discussion Topics / Grooming
- Sprint 49 focus
     - Getting changes prepped for Discharge Upgrade wizard review board links (need to meet with Margarete and will need content for either scenario)    **_- OR -_**
     - Making changes to Find a VA Form while Mark implements popularity logic
            - "Related To" links (needs design intent review)
            - PDF download info (needs design intent review)
            - Design UI consistency changes
 - Design Pattern for Search - template has just been finalized.   https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new?assignees=&labels=&template=documentation_template.md      
 - Marci has contacted Boris Ning to see if we can run his PDF script that detects dynamic XFA.  He has given us a link to the script and documentation. https://github.com/fartbagxp/pdf-xfa-detection
 - Find a Form batch 1 copy - check on the status.  
### Discussion Topics / Grooming
- **Find a Form:** Search changes on Staging as of 1:15 PM.  Zach & Mark are debugging.  Should we de-scope having Mark attempt to implement the form number correction Zach already has in the FE, use that for now, and have Mark add to the API in another iteration?  I'm concerned about too many changes at once.
- **Find a Form** - I pinged Randi and Beth earlier about Content recommendation re: word(s) to use for the "relevance" sort option.  Request has been open for 2 weeks.
- **Search UI Consistency** - We are putting this in the icebox because there is other work Cassandra and Megan can be doing until the Design System team has a pattern template ready.
- **Discharge Upgrade Wizard** #23988 - we do still need to change the PDF URL; I'm still seeing a problem like John was.
- **Discharge Upgrade Wizard** - Need to talk with stakeholder that requested PDF to eApplication link for Air Force personnel.  Content has asked if we still plan to provide the PDF link and we don't have an answer.    This is a blocker for the Content request, which in turn is a blocker for a UI design.  Alexander Reber is our contact on this -- Marci will schedule a meeting - Randi Hecht also.  Scope may also expand if this is not a unique page for USAF.
- **Yellow Ribbon** - Pulled two issues out of Icebox.  I will contact Janice Fisher at VBACACO to get an ETA on the PDF download and school data update.  #16404 - tent. Sprint 48 #16500 - tent. Sprint 50
- Maybe we pivot to YR for bit once this search change is in place?
- 
 
#### New GH Issues
- #24578 
- #24616

<hr>

### Discussion Topics / Grooming
- Find a Form: Mark Viterna is going to prioritize the Spanish language URLs over the search prioritization, which will probably push our testing into Sprint 47.  Platform is also struggling to get SOCKS working on my machine; probably spent four hours between yesterday and today.
- Find a Form: Some additional touch points for the FE part of search prioritization due to the need for a new sort option (now touching Content, Midpoint Review, Staging Review & Contact Center)
- Search UI Consistency: Follow-up meeting next Monday to cover items needing further Design/IA/Accessibility direction.
- Discharge Upgrade Wizard: Alexander at USAF doesn't currently have a timeline on when they will have a cert so we can point to their eApplication, but we should prob have Cassandra mockup what that's going to look like next sprint.
- Discharge Upgrade Wizard - any new tasks arising out of testing upgrades/refactoring?  Refactor may roll over.
- Ping Mark/Nichole re: timeline on Spanish URL fix

#### New GH Issues
- [x] New issue for Discharge Upgrade Wizard - changing another PDF link on https://www.va.gov/discharge-upgrade-instructions/request-dd214
- [X] Already entered a Content request for verbiage for the select #23912
- [X] Add issue for internal testing of search logic on Review/Staging #22866
- [ ] Add issue for FE work to add new sort option [**Combine with Feature Flag??**]
- [x] Add issue for Midpoint review #23989
- [X] Add issue for Staging review #23992
- [X] Add issue for updating Contact Center documentation #23995
- [X] Add issue for Contact Center review #23993
- [x] Add issue for Cassandra to mock up verbiage where we change Discharge Upgrade Wizard PDF link to eApplication link #23591
- [x] New issue for content request for Discharge Upgrade Wizard (🔴 blocker for #23591)
- [x] Add Shawna to the work session Mon
- [x] Backlog item to link Form  name on the detail page when there are related forms.

<hr>

#### New GH Issues

- Discharge Wizard #21944 Discharge Upgrade Wizard - change XFA based PDF link (bug fix)
- Discharge Wizard #23155 Testing Upgrades
- Discharge Wizard #23156 Organize Application
-  Discharge Wizard #23157 Possible write code as needed (may become epic)
- Discharge Upgrade Wizard #23071 - Change PDF link to eApplication URL for Air Force respondents
     - Create new ticket for UI changes
     - Create new ticket for content changes

### Prioritization
- Priority of Yellow Ribbon 508 issues vs. Discharge Upgrade Wizard
- Risk of linking Discharge Upgrade Wizard to page that throws privacy error

### Discussion Topics / Grooming

- Next steps for search improvements - can it be done with Lighthouse or too complex?  Do we need to seek Premal's input?
- Previously-blocked language-related 508 Issue moved to Level 1  #14432
- New GH issues
       - #22458 
       - #22453 
        - #22452
        #22460 

<hr>

## Discussion Topics / Grooming

- Search string matching improvements  EPIC #17244
    - Collaborate with S&D team (meeting next week) to see if we can leverage bet bets/type ahead
    - Form aliases will apply to both tools; don't need to maintain in multiple places.
- User Testing / Recommendations for Find a VA Form search results
    - Get user feedback/identify pain points in current search tool
    - Once Cindy joins, we will meet to discuss next steps
- User Testing / Recommendations for Yellow Ribbon
    - Get user feedback/identify pain points in current search tool
    - Once Cindy joins, we will meet to discuss next steps
- Related To bug - Zach will use the existing issue #19561
<hr>

### Discussion Topics/Grooming 
- [ ] Agree on definition of done
- [ ] Review slide deck from VA Forms Stakeholder's meeting 
- [ ] Roadmap

Notes/Next Steps: 

### Find a VA Form: Improve Search String Comparison
- [ ]  EPIC #17244

Notes/Next Steps:
- Workshop this afternoon

### Yellow Ribbon

Notes/Next Steps: 
- [ ] Item #24578 ##  :star: Wednesday, April 28, 2021 (mid-Sprint 46)
### Discussion Topics / Grooming
- Find a Form: Mark Viterna is going to prioritize the Spanish language URLs over the search prioritization, which will probably push our testing into Sprint 47.  Platform is also struggling to get SOCKS working on my machine; probably spent four hours between yesterday and today.
- Find a Form: Some additional touch points for the FE part of search prioritization due to the need for a new sort option (now touching Content, Midpoint Review, Staging Review & Contact Center)
- Search UI Consistency: Follow-up meeting next Monday to cover items needing further Design/IA/Accessibility direction.
- Discharge Upgrade Wizard: Alexander at USAF doesn't currently have a timeline on when they will have a cert so we can point to their eApplication, but we should prob have Cassandra mockup what that's going to look like next sprint.
- Discharge Upgrade Wizard - any new tasks arising out of testing upgrades/refactoring?  Refactor may roll over.
- Ping Mark/Nichole re: timeline on Spanish URL fix

#### New GH Issues
- [x] New issue for Discharge Upgrade Wizard - changing another PDF link on https://www.va.gov/discharge-upgrade-instructions/request-dd214
- [X] Already entered a Content request for verbiage for the select #23912
- [X] Add issue for internal testing of search logic on Review/Staging #22866
- [ ] Add issue for FE work to add new sort option [**Combine with Feature Flag??**]
- [x] Add issue for Midpoint review #23989
- [X] Add issue for Staging review #23992
- [X] Add issue for updating Contact Center documentation #23995
- [X] Add issue for Contact Center review #23993
- [x] Add issue for Cassandra to mock up verbiage where we change Discharge Upgrade Wizard PDF link to eApplication link #23591
- [x] New issue for content request for Discharge Upgrade Wizard (🔴 blocker for #23591)
- [x] Add Shawna to the work session Mon
- [x] Backlog item to link Form  name on the detail page when there are related forms.

<hr>

### Technical Debt Opportunities 
- Find a VA Form 508 Defects: #19229
- Yellow Ribbon 508 Defects: #19205

Notes/Next Steps: 

### Team Clarity Requests: 

Notes/Comments:

<hr>
